test/integration/puppetlabs/services/jruby_pool_manager/jruby_internal_test.clj

```diff
@@ -30,11 +30,28 @@
   (testing "setting plumbed into jruby container for"
     (let [pool (JRubyPool. 1)
           config (jruby-testutils/jruby-config
-                  {:compile-mode :jit})
+                  {:compile-mode :jit
+                   :compat-version 2.0})
           instance (jruby-internal/create-pool-instance! pool 0 config #())
           container (:scripting-container instance)]
       (try
         (is (= RubyInstanceConfig$CompileMode/JIT
             (.getCompileMode container)))
+        (is (.is2_0 (.getCompatVersion container)))
         (finally
           (.terminate container))))))
+
+(deftest get-compat-version-test
+  (testing "returns correct compat version for SupportedJrubyCompatVersions enum"
+    (is (not (.is2_0 (jruby-internal/get-compat-version "1.9"))))
```

Comment:
Could this be `(is (.is1_9...`?

Huh, looks like it's a little funky, from the jruby code for `is1_9`
`return this == RUBY1_9 || this == RUBY2_0;`

Maybe check that `is1_9` is true and `is2_0` is false too?